azure-quantum/azure/quantum/workspace.py

```diff
@@ -33,27 +34,17 @@ def sdk_environment(name):
 
 ## Settings based on environment variables:
 BASE_URL_FROM_ENV = os.environ['AZURE_QUANTUM_BASEURL'] if 'AZURE_QUANTUM_BASEURL' in os.environ else None
-if  sdk_environment('dogfood'):

```

Comment:
Are dogfood and canary no longer included? If so, we should make sure to remove from IQ# as well.

Reply:
I need to check this; will put them back in for now.